pkgs/by-name/su/sunxi-tools/package.nix

```
@@ -6,17 +6,18 @@
   dtc,
   libusb1,
   zlib,
+  unstableGitUpdater,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation {
   pname = "sunxi-tools";
-  version = "unstable-2021-08-29";
+  version = "unstable-2024-10-13";
```

```suggestion
  version = "0-unstable-2024-10-13";
```
New policy, all versions must start with a digit.

---

Updated!